buildSrc/src/main/java/io/reactor/gradle/CustomVersionPlugin.java

```diff
@@ -24,8 +24,8 @@
 
 /**
  * Looks for a {@code -PversionBranch=foo} type of property and uses that as an additional
- * suffix between the patch number and the snapshot qualifier ({@code .BUILD-SNAPSHOT}), in
- * order to generate branch-specific snapshots that can be used to eg. validate a PR.
+ * suffix after the snapshot qualifier (either {@code .BUILD-SNAPSHOT} or {@code -SNAPSHOT}),
```

Comment:
can be adjusted to just "ends with -SNAPSHOT"

Reply:
yeah I guess we can fully drop support of the `.BUILD-SNAPSHOT` in this branch.